lib/Wasmtime.hs

```diff
@@ -1318,9 +1336,11 @@ peekTableVal :: Ptr C'wasmtime_val_t -> IO TableValue
 peekTableVal val_ptr = do
   k <- peek kind_ptr
```

Comment:
where does kind_ptr come from? Am I confused or is this a typo?

Reply:
It's [defined in the where clause](https://github.com/dfinity/wasmtime-hs/blob/cd57ad1275f52eff15f17da2a833176ddb940e9a/lib/Wasmtime.hs#L1349).

Reply:
thanks, the github view obfuscated that :D